packages/stories/package.json

```diff
@@ -40,8 +42,10 @@
     "@theforeman/builder": "^3.10.1",
     "@theforeman/vendor": "^3.10.1",
     "babel-loader": "^8.0.6",
+    "babel-plugin-react-docgen": "^3.2.0",
     "commander": "^4.0.1",
     "node-sass": "^4.5.0",
+    "react-is": "^16.12.0",
```

Comment:
I don't see any use for it, is it needed?

Reply:
Yes, see: https://github.com/storybookjs/storybook/blob/c0f3f71e0307d27d72bd142b9f9ad8413a675731/addons/docs/react/README.md#mdx